setup.py

```diff
@@ -110,15 +135,15 @@
     #
     #   py_modules=["my_module"],
     #
-    #packages=find_packages(exclude=['contrib', 'docs', 'tests']),  # Required
+    # packages=find_packages(exclude=['contrib', 'docs', 'tests']),  # Required
     packages=find_packages(),
     # This field lists other packages that your project depends on to run.
     # Any package you put here will be installed by pip when your project is
     # installed, so they must be valid existing projects.
     #
     # For an analysis of "install_requires" vs pip's requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
-    install_requires=['scipion-em'],  # Optional
+    install_requires=['scipion-em', 'emtable'],  # Optional
```

Comment:
Why not linkng this with requirements.txt as we've done in others?

Reply:
there was no requirements.txt before the PR, so I didnt make one. But if you prefer creating a new file instead of 1 line of code, sure :)

Reply:
done!